### ISSUE TITLE
Render bug with items in crates depending on positive/negative X/Z coordinates

### ISSUE THREAD

**Issue by mateon1** [Funwayguy/BarrelsDrumsStorageAndMore#24]
Using version 0.0.24 with the SkyFactory 4 modpack.

The sky_orchards mod's saplings (and possibly other items, but I haven't found other examples yet) get rendered incorrectly when a barrel is placed on a negative Z coordinate. The incorrect rendering looks like the item is overlapped with a mirror image of itself, causing Z-fighting to occur.

If the barrel is on a +X/-Z coordinate, the item is always rendered incorrectly, when the barrel is placed on a -X/-Z coordinate, the item is rendered incorrectly only if the player is on a positive X coordinate.

Screenshots:
https://i.imgur.com/EhKrw5s.png - Player on positive X and Z. All sapling items rendered incorrectly.
https://i.imgur.com/ScYL7A3.png - Player on negative X, positive Z. Saplings in barrels on +X/-Z coordinates rendered incorrectly, the rest are fine.

Regardless of the player's position, items are rendered correctly in barrels with a positive Z coordinate.

**Comment by mateon1** [comment]
Looks like this also affects leaf blocks, but the effect is much more subtle, looks like the "wrongly rendered" version also renders the back faces of the leaf blocks.

**Comment by Funwayguy** [comment]
I don't think this is an issue with the crates. Those saplings also have issues when rendered in 3rd person last I checked. Considering no other modded block or fluid I've tested has had any issues like this, I'm inclined to believe this to be an issue on Sky Orchards' side.

**Comment by mateon1** [comment]
I can confirm this also happens with Rustic apple leaves and Vanilla acacia leaves. The issue does not happen for any vanilla saplings nor Rustic apple saplings.
Seems that Sky Orchards saplings are just very noticeable when they trigger this bug.
